src/language-css/printer-postcss.js

```diff
@@ -915,7 +917,17 @@ function genericPrint(path, options, print) {
               path.map((childPath, index) => {
                 const child = childPath.getValue();
                 const isLast = index === node.groups.length - 1;
-                const printed = [print(), isLast ? "" : ","];
+                const hasComma = () =>
+                  Boolean(
+                    child.source &&
+                      options.originalText
+                        .slice(locStart(child), locStart(node.close))
+                        .trimEnd()
+                        .endsWith(",")
+                  );
```

Comment:
Ideally we need to use `getNextNonSpaceNonCommentCharacterIndexWithStartIndex` like this:

```js
                const hasComma = () => {
                  const index =
                    child &&
                    getNextNonSpaceNonCommentCharacterIndexWithStartIndex(
                      options.originalText,
                      locEnd(child)
                    );
                  return index && options.originalText[index] === ",";
                };
```

But, `locEnd` for `--baz` in `var(--baz,)` returns unexpected value

Reply:
https://github.com/prettier/prettier/pull/13321 is also blocked by it.